src/components/Settings/Settings.styles.js

```diff
@@ -0,0 +1,53 @@
+import styled from '@emotion/styled';
+
+import Form from 'components/Form';
+
+export const SettingsContainer = styled.div`
+  background-color: ${(props) => props.theme.color.gray200};
+  display: flex;
+  flex-direction: column;
+  min-height: inherit;
+`;
+
+export const Wrapper = styled.div`
+  align-items: center;
+  background-color: ${(props) => props.theme.color.white};
+  border-radius: ${(props) => props.theme.borderRadius.md};
+  display: flex;
+  flex-direction: column;
+  margin: 10px auto;
+  padding: 2rem;
+  width: auto;
+`;
+
+export const SettingsTitle = styled.h2`
+  align-self: center;
+  color: ${(props) => props.theme.color.gray700};
+  font-size: ${(props) => props.theme.fontSize.x3l};
+  font-weight: 300;
+  grid-area: title;
+  margin-bottom: 2.5rem;
+  text-align: center;
+`;
+
+export const SuccessText = styled.p`
+  color: ${(props) => props.theme.color.green700};
+`;
+
+export const FormBody = styled(Form)`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-top: 1.5rem;
+  width: fit-content;
+`;
+
+export const FormButton = styled(Form.Button)`
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  width: 16rem;
+
+  &:hover {
+    background-color: ${(props) => props.theme.color.indigo600};
+  }
```

Comment:
Why did you override de this? You did it in purpose? I'm asking because it nullifies the hover effect

Reply:
yep haha. it didnt look good, but i can bring it back

Reply:
Up to you, I just wanted to verify haha